HISTORY.md

```diff
@@ -22,6 +22,7 @@ All notable changes to this project will be documented in this file.
   * Apply latest code style guide (via [#1170], [#1181])
 * Dependencies
   * Support `libxmljs2@^0.35` (via [#1173])
+  * Support `libxml2-wasm@^0.41` as an alternative for `libxmljs2` (via [#1184])
```

Comment:
Since a release of 7.0.0in the meantime, and a forward-merge,
this needs to be updated to be in the `unreleased` category.